pkg/remediator/crashloopbackoff_rescheduler_test.go

```diff
@@ -122,17 +124,21 @@ func (suite *TestCrashLoopBackOffReschedulerSuite) testRemediator() {
 	ctx, cancel := context.WithCancel(suite.ctx)
 	cancel() // cancel first so we can just run once and exit
 
+	suite.mockClient.EXPECT().GetSharedInformerFactory(suite.logger, "").Return(suite.testGetInformerFactory(), nil).Times(1)
 	crashLoopRemediator, err := suite.testGetRemediator()
 	assert.Equal(suite.t, err, nil)
 	assert.Assert(suite.t, crashLoopRemediator != nil)
 	crashLoopRemediator.Run(ctx, nil)
 }
 
+func (suite *TestCrashLoopBackOffReschedulerSuite) testGetInformerFactory() informers.SharedInformerFactory {
```

Comment:
not a test -> no test prefix ?
```suggestion
func (suite *TestCrashLoopBackOffReschedulerSuite) getInformerFactory() informers.SharedInformerFactory {
```

Reply:
still needs a rename ? `newInformerFactory `